### DISCUSSION TITLE
How can I cross-reference non-template elements?

### DISCUSSION THREAD

**Discussion by karl-an** [quarto-dev/quarto-cli#6796] [Q&A] [answered]
### Description

I have some exercises in my lecture slides that I want to number across slides and cross-reference.
I would like to 

```
## Task {#task-first_task}

Open R.



## Task {#task-second_task}

Close R.

```

and later cross-reference it as 
Please solve `[@task-first_task]` first. 

How is that possible? 

Thank you and best regards

**Comment by cderv** [comment, marked as answer]
Possibly #4944 will allow this. cc @cscheid

**Reply by cscheid** [reply]
Yes. Keep an eye specifically on "control numbering of anchor crossrefs".
